fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1700,6 +1729,7 @@ Future<Void> ReadYourWritesTransaction::watch(const Key& key) {
 	if (key.size() > (key.startsWith(systemKeys.begin) ? CLIENT_KNOBS->SYSTEM_KEY_SIZE_LIMIT : CLIENT_KNOBS->KEY_SIZE_LIMIT))
 		return key_too_large();
 
+	approximateSize += key.expectedSize();
```

Comment:
I think the cost here was from reading a key, right? I think then that this should move to the `ACTOR` implementation of this function and only add the size of the conflict range (`2*size+1`) if we are bypassing RYW.

Reply:
Done.